tests/valgrind/tleak_arc.nim

```diff
@@ -0,0 +1,14 @@
+discard """
+valgrind: true
+cmd: "nim $target --gc:arc -d:useMalloc $options $file"
+exitcode: 1
+outputsub: "   definitely lost: 7 bytes in 2 blocks"
+disabled: "freebsd"
+disabled: "macosx"
+disabled: "openbsd"
+disabled: "windows"
+disabled: "32bit"
```

Comment:
You don't need that,:

https://github.com/nim-lang/Nim/blob/9a110047cbe2826b1d4afe63e3a1f5a08422b73f/testament/specs.nim#L307-L316

Reply:
We actually do in this case, since otherwise the test will be run without valgrind on these platforms and thus we won't get the error message/exit code.

Reply:
I think we should add a "valgrind: leak" attribute.

Reply:
Doesn't help here, but had to be done for the failing tests anyways.